### DISCUSSION TITLE
Version beta.44 / beta.45 - was ist da Neu?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2693] [General]
Sonst hast du immer zum Issue einen Hinweis reingestellt.

**Comment by Zefau** [comment]
Die Releases werden wieder gelöscht. Der Workflow zur Android App Erzeugung geht aktuell auf die Bretter bei neuen Releases. Bekomme das nicht gefixed, daher die ganzen Releases.
